pyaedt/application/DesignSolutions.py

```diff
@@ -0,0 +1,678 @@
+import copy
```

Comment:
Since this is a new file, we might as well start refactoring our module names to use snakecase to be PEP8 compliant.

Reply:
Correct. I've renamed it to design_solutions.py